stix2/exceptions.py

```diff
@@ -49,3 +49,10 @@ class ImmutableError(STIXError, ValueError):
 
     def __init__(self):
         super(ImmutableError, self).__init__("Cannot modify properties after creation.")
+
+
+class VersioningError(STIXError, ValueError):
+    """Execption while using the Versioning API"""
+
+    def __init__(self, msg):
```

Comment:
Maybe split this into separate exceptions? `AlreadyRevokedError` and `UnmodifiableFieldError`? The latter could take the list of fields as an argument, so we can check that without needing to do exact string matching.